package.json

```diff
@@ -92,8 +93,8 @@
     "ts-loader": "^5.3.0",
     "typescript": ">=2.9",
     "url-loader": "^4.0.0",
-    "vue": "^2.3.4",
-    "vue-loader": "^15.0.11",
+    "vue": "^3.0.0beta.9",
+    "vue-loader": "^16.0.0-alpha.1",
```

Comment:
With `^2.3.4 || ^3.0.0beta.9`, yarn ALWAYS prefers the stable version 2. So, I've added the `-beta`-alpha` versions only here so that they will install. Then in `scripts/force-lowest-dependencies.js`, I manually force vue2 and vue-loader 15. It's super hacky - would love if someone knows a way for me to use `^2.3.4 || ^3.0.0beta.9` but tell yarn to install the beta v3.